server/src/main/scala/de/frosner/broccoli/models/ParameterType.scala

```diff
@@ -10,4 +10,6 @@ object ParameterType extends Enum[ParameterType] with PlayJsonEnum[ParameterType
   val values: immutable.IndexedSeq[ParameterType] = findValues
   case object Raw extends ParameterType
   case object String extends ParameterType
+  case object Integer extends ParameterType
+  case object Decimal extends ParameterType
```

Comment:
Why do we differentiate between integer and decimal? In the end what gets serialized is anyway going to be a JS number. What is the type backing up `JsNumber` from play? `BigDecimal`? `Double`? Should we use that one as the numeric type?

Reply:
If you have things like CPU then decimals do not make sense. There is no 1 to one mapping between JSON types and the types we define here. This was the idea. But I can combine them if you think it makes more sense.

Reply:
I think we shouldn't overcomplicate things. If Nomad fails when specifying 200.4 MHz CPU then we should have ints to fail fast. If Nomad doesn't care, I'd prefer to merge them especially as in JSON it doesn't make a difference.

What do you think @Gerrrr, @aurelianrl ?